.github/workflows/stale.yml

```diff
@@ -0,0 +1,19 @@
+name: Close Stale Issues
+uses: actions/stale@1.0.0
+  with:
+    # Token for the repo. Can be passed in using {{ secrets.GITHUB_TOKEN }}
+    repo-token: 
+    # The message to post on the issue when tagging it. If none provided, will not mark iusses stale.
```

Comment:
Seems like your required to have token for repo and perhaps changing this to would fix it proceeded by adding a github secret to repo.
```
repo-token:  {{ secrets.GITHUB_TOKEN }}
```